src/App.js

```diff
@@ -65,6 +62,20 @@ export default class App extends Component {
     setLoadingEntity: (loadingEntity) => {
       this.setState({ loadingEntity });
     },
+    setSetting: (settingKey, settingValue) => {
+      let states = {
+        settings: { ...this.state.settings, [settingKey]: settingValue },
+      };
```

Comment:
const settings = { ...this.state.settings, [settingKey]: settingValue },

Reply:
Ok, I just put it there for the console.log, I guess I can remove it